src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -164,30 +158,30 @@
     static HttpTransport testTransport;
 
     /**
-     * The transport loaded with a service account secret.
-     * Marked static for https://jenkins.io/blog/2018/03/15/jep-200-lts/#after-the-upgrade
-     * Will get repopulated on restart
+     * The transport loaded with a service account secret. Marked static for
+     * https://jenkins.io/blog/2018/03/15/jep-200-lts/#after-the-upgrade Will get
+     * repopulated on restart
      */
-    private static HttpTransport transport;
-    
+    private transient HttpTransport transport;
```

Comment:
Yeah per you comment `Turn the jvmDefaultKeystoreTransport into an instance variable. When used as static, it was reusable between users, making the connection conveying the credentials of previously attempted users` 

I think I might see how there was a problem, though not necessarily what you have conveyed. 

I would like to walk through some of the details of how you landed there, and want to make sure you went through the same path to this conclusion.

Specifically:
a) there is only one security realm per JVM
b) but given the crappy Jenkins API, we have to recreate that instance each time we add users
c) that said, the transport does not have user specific elements to it .... if you discovered there is, elaborate as to why ... I don't see it
d) but maybe certs rotated and that somehow mess things UP, as the two transport choices needed to be regenerated ?  though they are re-gened in the ctor and populateDefaults